provider/validate.go

```diff
@@ -62,6 +62,11 @@ func (s *RawProviderServer) ValidateResourceTypeConfig(ctx context.Context, req
 	rawManifest := make(map[string]tftypes.Value)
 	err = manifest.As(&rawManifest)
 	if err != nil {
+		if err.Error() == "unmarshaling unknown values is not supported" {
+			// likely this validation call came to early and the manifest still contains unknown values
+			// bailing out witout error to allow the resource to be completed at a later stage
```

Comment:
Looks good! Just a suggestion for the comment:
```suggestion
			// Likely this validation call came too early and the manifest still contains unknown values.
			// Bailing out without error to allow the resource to be completed at a later stage.
```